src/com/dakl/Tests.java

```diff
@@ -1,5 +1,6 @@
 package com.dakl;
 
+// REVIEW : separate test classes into a different source root
```

Comment:
It is typically recommended to separate the test code into a different directory / source root so that whatever compiles and packages your code into an executable has an easy way to differentiate between test and production code (whereas test code shouldn't be shipped)